src/main/java/com/google/sps/data/Contest.java

```diff
@@ -75,9 +76,9 @@ public static Contest fromJSONObject(DatastoreService datastore, JSONObject cont
   // Creates a new Contest object by using the propperties of the provided
   // contenst entity
   public static Contest fromEntity(Entity entity) {
-    HashSet<Long> candidates = new HashSet<>();
+    Set<Long> candidates = new HashSet<>();
```

Comment:
nit: ImmutableSet.of()

Can just leave it as an empty set.

Reply:
done